### DISCUSSION TITLE
I want to include E-Mail in to WebKul WhatsApp Bot Popup.

### DISCUSSION THREAD

**Discussion by Officesleek** [breezefront/community#303] [1. Help]
How can I include Email in the following Buy on WhatsApp popup?
![image](https://github.com/user-attachments/assets/75b37990-23b4-4dbf-94c3-1cc13a3f9d32)

**Comment by Officesleek** [comment]
Anyone can help here please?

**Reply by vovayatsyuk** [reply]
Is it visible without Breeze?

**Reply by Officesleek** [reply]
No

**Reply by vovayatsyuk** [reply]
You should contact Webkul developers. It's their module so they will give you the best answer.
